src/index.js

```diff
@@ -843,6 +843,7 @@ export class TextField extends MaterialComponent {
 		if (input && input.value && input.value!==this.props.value) {
 			input.value = this.props.value;
 		}
+		input.setCustomValidity(this.props.errorMessage || "");
```

Comment:
Everything looks A+, but I think it'd be best to wrap this in a feature detection.

```js
if (input.setCustomValidity) {
  input.setCustomValidity(this.props.errorMessage || "");
}
```